code/modules/power/power_machine.dm

```diff
@@ -87,23 +139,16 @@
 		return ..()
 
 
+/obj/machinery/power/proc/disconnect_terminal() // machines without a terminal will just return, no harm no fowl.
+	return
+
 ////////////////////////////////////////////////
 // Misc.
 ///////////////////////////////////////////////
 
 
-// return a knot cable (O-X) if one is present in the turf
-// null if there's none
-/turf/proc/get_cable_node()
-	if(!can_have_cabling())
-		return null
-	for(var/obj/structure/cable/C in src)
-		if(C.d1 == NO_DIRECTION)
-			return C
-	return null
-
-/area/proc/get_apc()
-	for(var/thing in GLOB.apcs)
-		var/obj/machinery/power/apc/APC = thing
-		if(APC.apc_area == src)
-			return APC
+/// Electricity fucking hurts! Let's make sure players understand that
+/obj/machinery/power/proc/attack_zap_check(mob/living/user)
```

Comment:
Why bother checking the type if you're gonna return the same thing?

Reply:
need this to return these values while in testing, I still haven't fully implemented zapping yet